### ISSUE TITLE
Rendering image to drawable: no texture found on color attachment 0. This could happen when the drawable size is less than 16x16 p

### ISSUE THREAD

**Issue by sagarthecoder** [MetalPetal/MetalPetal#373]
I am using **MTIThreadSafeImageView**.
```
threadSafeImageView.image = mtiImage
```
Sometimes app is crushed and shown this error. for the same case sometimes it works without error.
How to solve?
Error is `Rendering image to drawable: no texture found on color attachment 0. This could happen when the drawable size is less than 16x16 p`

**Comment by ChaseStas** [comment]
threadSafeImageView probably has frame size zero at the moment when you set image to it (I has same issue)

**Comment by sagarthecoder** [comment]
@ChaseStas Yes, that was the issue. When I finally showed it after a delay, it worked.
